### DISCUSSION TITLE
How does Tasmota work directly with Ethernet?

### DISCUSSION THREAD

**Discussion by pioueo** [arendst/Tasmota#20259] [Q&A] [answered]
Hi community,

Is there a way to access Tasmota WebUI directly without going through the Wifi configuration if ethernet communication is present (lan8720 for example)?

Also, how do I configure a fixed ethernet IP address in Tasmota32 sources files? I can see the Wifi settings but not the ethernet settings.

**Comment by sfromis** [comment, marked as answer]
When Ethernet is connected and enabled, I can access Tasmota by adding `-eth` to the normal hostname, which my router automatically provides DHCP+DNS resolution for. Thus I have no need for configuring a static IP address, and if wanting something like that, I'd much prefer fixed address assignments in DHCP.

Still, among the Tasmota commands there are specific commands for Ethernet, including IP address assignment.
https://tasmota.github.io/docs/Commands/#ethernet
And any command can be included at build time using `USER_BACKLOG`.

**Reply by pioueo** [reply]
@sfromis Thank you very much. It works very well. Tasmota is really great!
